utility/src/main/java/org/oppia/android/util/metriccollection/UiSpecificMetrics.kt

```diff
@@ -0,0 +1,8 @@
+package org.oppia.android.util.metriccollection
+
+import org.oppia.android.app.model.OppiaMetricLog
+
+interface UiSpecificMetrics {
```

Comment:
For this & the other metrics classes:
- There are some inconsistencies in names (most are called 'metrics' but one is 'metricutils')
- These names conflate a bit with their proto counterparts
- I'm a bit confused why these can't just be part of the controller, and have the controller fill in the information for a particular log. It seems logically simpler to consolidate to a single source of truth that's managing the data within the scope of where it's needed (logged events)

Reply:
- It was called metricUtils because other files were related to a specific type of looggable metric (like app startup, etc) but metricUtils was related to the other information that has to be logged along with the luggable metric (like isAppInForeground, etc). 
- I've put all of them together in PerformanceMetricsUtils and then used these util functions in both the controller and the logger. It didn't make sense for any of them to be in the controller, design wise -- since they are utilities and also because defining these functions in the controller, might lead to circular dependency issues.

Does that help?

Reply:
I think it's cleaner now, but why does anything other than the controller need to depend on these utilities?

Reply:
Apart from the controller, the logger also needs to depend on these utilities to pack data into proto definitions. Essentially, the data is of two types: event data and environment data. For example, when an apk size metric is logged -- apk size is event data and isAppInForeground is environment data. Event data is packed at the logger as part of LoggableMetric while the environment data is packed by the controller as part of OppiaMetricLog. Does that make sense?

Reply:
Yep. So I think it's fine to have a central utility, but let's try to use a clearer name than just 'Utils'. Perhaps: 'PerformanceMetricsAssessor' would be clearer?

Reply:
done.